frontend/src/components/userProfile/UserProfileForm.tsx

```diff
@@ -35,106 +37,191 @@ const UserProfileForm = ({
+            value={fullName}
+            onChange={(e) => setFullName(e.target.value)}
+          />
+          <Heading marginTop="24px" size="sm">
+            Email*
+          </Heading>
+          <Input
+            type="email"
+            value={email}
+            onChange={(e) => setEmail(e.target.value)}
+          />
+          <Heading marginTop="24px" size="sm">
+            Educational qualifications*
+          </Heading>
+          <Textarea
+            height="150px"
+            type="qualifications"
+            onChange={(e) => setEducationalQualification(e.target.value)}
+          />
+          <Heading marginTop="24px" size="sm">
+            Language experience*
+          </Heading>
+          <Textarea
+            height="150px"
+            type="experience"
+            onChange={(e) => setLanguageExperience(e.target.value)}
+          />
+        </FormControl>
+        {isSignup && (
+          <>
+            <Heading marginTop="24px" size="sm">
+              Select language*
+            </Heading>
+            <Text marginBottom="12px">
+              Select the language that you would like to start with. Additional
+              languages can be added after signing up.
+            </Text>
+            <Box width="60%" marginBottom="200px">
+              <Select
+                placeholder="Select language"
+                options={languageOptions}
+                getOptionLabel={(option: any) => `
+                  ${convertLanguageTitleCase(option.value || "")}
+                `}
+                components={{ DropdownIndicator }}
+              />
+            </Box>
+          </>
+        )}
+      </Flex>
+      <Flex direction="column" flex={1} marginLeft="60px" marginTop="75px">
+        <Heading size="sm">Upload your CV (optional)</Heading>
+        <Flex
+          align="center"
+          background="#e8f0f6"
+          border="2px"
+          // this is blue.100 at 10% opacity
+          borderColor="rgba(29, 108, 165, 0.1)"
+          direction="column"
+          height="250px"
+          justify="center"
+          maxWidth="80%"
+          onDragEnter={(e: any) => onPreventDefault(e)}
+          onDragLeave={(e: any) => onPreventDefault(e)}
+          onDragOver={(e: any) => onPreventDefault(e)}
+          onDrop={(e: any) => onFileDrop(e)}
+          rounded="md"
+        >
+          <IconButton
+            aria-label="Folder icon"
+            background="transparent"
+            color="blue.100"
+            icon={<Icon as={MdFolder} width={16} height={16} />}
+            marginBottom="16px"
+            width="fit-content"
+          />
+          <Text>Drag & drop your file here (.PDF, .docx)</Text>
+          <FormControl width="160px">
+            <FormLabel cursor="pointer" htmlFor="browse-files" marginTop="16px">
+              <Button colorScheme="blue" pointerEvents="none">
+                Browse Files
+              </Button>
+            </FormLabel>
```

Comment:
I wrapped a `Button` component in a `FormLabel` for styling, but I can just style the `FormLabel` to look like a button if that's better

Reply:
looks good to me